packages/modul-components/src/components/rich-text-editor/rich-text-editor.ts

```diff
@@ -99,6 +99,21 @@ export class MRichTextEditor extends ModulVue implements InputManagementData, In
     })
     public lastHeaderLevel: number;
 
+    @Prop({ default: false })
+    public showCharCounter: boolean;
+
+    @Prop()
+    public charCounterMax: number;
+
+    @Prop({ default: false })
+    public showParagraphAlignementButtons: boolean;
+
+    @Prop({ default: false })
+    public showUnderlineButton: boolean;
+
+    @Prop({ default: false })
+    public showStrikeThroughButton: boolean;
```

Comment:
J'aurais tendance à ajouter ces options dans l'enum MRichTextEditorOption.

Reply:
@jpguilmette est venu me voir concernant ça , le problème avec l'enum vs props c'est que l'enum fit pas vraiment avec la documentation dans website et aussi on doit ajouter des if partout dans le code du genre `this.options.includes(MRichTextEditorOption.IMAGE_HIDE_FLOAT_LAYOUT)` 

On va en discuter lundi a la rencontre modul @jipigi @cabob  @jpguilmette

Reply:
Quelle que soit l'orientation technique, il faut ne conserver qu'une seule façon de faire. Est-ce qu'on fait le BC de retirer l'enum et le remplacer par deux props showImageButtun et showImageFloatLayout ?

Reply:
J'irais dans cette direction. Soit retirer Options et les convertir en props. Ça serait plus proche du reste du comportement de modul